tests/test_cnn/test_revert_syncbn.py

```diff
@@ -0,0 +1,59 @@
+import os
+import platform
+
+import numpy as np
+import pytest
+import torch
+import torch.distributed as dist
+
+from mmcv.cnn.bricks import ConvModule
+from mmcv.cnn.utils import revert_sync_batchnorm
+
+if platform.system() == 'Windows':
+    import regex as re
+else:
+    import re
+
+
+def test_revert_syncbn():
+    conv = ConvModule(3, 8, 2, norm_cfg=dict(type='SyncBN'))
+    x = torch.randn(1, 3, 10, 10)
+    # Expect a ValueError prompting that SyncBN is not supported on CPU
+    with pytest.raises(ValueError):
+        y = conv(x)
+    conv = revert_sync_batchnorm(conv)
+    y = conv(x)
+    assert y.shape == (1, 8, 9, 9)
+
+
+def test_revert_mmsyncbn():
+    if 'SLURM_NTASKS' not in os.environ or int(
+            os.environ['SLURM_NTASKS']) != 4:
```

Comment:
Are 4 tasks necessary here? Can one or two task make it work?

Reply:
We can make it as 2 to reduce the test cost. Originally set it as 4 for MMSyncBN is to test its behavior with different numbers of groups like 1, 2, 4. But they are unnecessary here.